lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1592,7 +1591,7 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :public => true,
           :type => String,
           :allowed_from_server => false,
-          :description => 'Specify a custom host name for [display in the New Relic UI](/docs/apm/new-relic-apm/maintenance/add-rename-remove-hosts#display_name).'
+          :description => 'Specify a custom host name for [display in the New Relic UI](/docs/apm/new-relic-apm/maintenance/add-rename-remove-hosts#display_name). Be aware that you cannot rename a hostname, so please rename carefully.'
```

Comment:
Maybe there's another way to word the warning here. Seeing "you can't rename" sitting right next to "rename carefully" stands out.

Reply:
Yes, I also find that confusing. I don't see a note like that on the docs site either, where is that coming from?

Reply:
I added this because at some point, someone found it relevant to add to our current `newrelic.yml`! We don't have to keep this, just as I was going through, this is the kind of thing that would be overwritten from changes we made to newrelic.yml overtime.

Reply:
I've reverted this change. We can address these specifics when we run this task against our current newrelic.yml 
https://github.com/newrelic/newrelic-ruby-agent/pull/2075/commits/5476256550f0b182eb525ff29d93e7bddc0ffcaf